astropy/utils/data.py

```diff
@@ -1069,6 +1073,10 @@ def download_file(remote_url, cache=False, show_progress=True, timeout=None,
         ``pkgname='astropy'`` the default cache location is
         ``~/.astropy/cache``.
 
+    http_headers : dict
+        HTTP request headers to pass into ``urlopen`` if needed. (These headers
+        are ignored if the ``remote_url`` is not http.)
```

Comment:
Actually it's `sources` that matters, not `remote_url`.

Reply:
Point taken, although it's confusing to work because `remote_url` does matter *if* `sources` is None.  Will try to wordsmith a bit.